content/docs/tasks/traffic-management/egress-tls-origination.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Perform TLS Origination for Egress Traffic
```

Comment:
Can you shorten the title to "TLS Origination for Egress Traffic"

Reply:
Yep, good idea, will do it

Reply:
@frankbu Done, shortened the title to "TLS Origination for Egress Traffic"